src/main/java/spotify/models/albums/AlbumFull.java

```diff
@@ -10,6 +10,30 @@
 
 import java.util.List;
 
+/**
+ * Class AlbumFull
+ * @package spotify.models.albums
+ * @author jzheng2017
```

Comment:
Should be consistent with other author formats

Reply:
OK. Thanks.